### PR TITLE
Implemented POST boost endpoint

### LINKED ISSUE #143: [TASK] POST /boost/:listing_id?packageID=package_id
### Description

Implement `POST /boost/:listing_id?packageID=package_id` endpoint

`:listing_id`: listingID of the listing that seller want to boost

`packageID=package_id`: packageID of the BoostPackage that seller picked. (this should be a query param)

### Acceptance Criteria:

- Should create a new entry on `Boost` table following the Class Diagram

### Link

**User Story**: #142 

**Pre-requisite**: #142

### PR DESCRIPTION
**Pull request by asifurrahman57**
- Implemented POST /boost/listing_id
- Requires authentication
- Return 404 Not Found if the listing_id does not exist in the DB
- Return 404 Not Found if the boost_package does not exist in the DB
- Return 200 when a listing is boosted

closes #143

### CODE REVIEW

File: fujiji-server/migration-scripts/example.sql [hunk 1]
```diff
@@ -1,4 +1,4 @@
-USE fujiji
+USe fujiji
```
Comment (wynonagcc) [inline fujiji-server/migration-scripts/example.sql:1]: I assume this is an accidental change, can you change it back to `USE`?

Comment (asifurrahman57) [reply fujiji-server/migration-scripts/example.sql]: I just fixed it

**Comment by devin-efendy** [comment]
Lgtm!

**Review by wynonagcc — APPROVED** [review]
Lgtm!

**Pull request merged**